src/coreclr/hosts/unixcorerun/corerun.cpp

```diff
@@ -484,3 +484,8 @@ int main(const int argc, const char* argv[])
                             managedAssemblyArgv);
     return exitCode;
 }
+
+int main(const int argc, const char* argv[])
+{
```

Comment:
Probably, but I didn’t want to add some kind of build define or create a separate file.  It doesn’t seem to hurt anything by being there in the sos plugin.

From: Jan Kotas [mailto:notifications@github.com]
Sent: Friday, April 24, 2015 6:49 PM
To: dotnet/coreclr
Cc: Mike McLaughlin
Subject: Re: [coreclr] Implemented hardware exception chaining. (#807)


In src/coreclr/hosts/unixcorerun/corerun.cpp<https://github.com/dotnet/coreclr/pull/807#discussion_r29096635>:

> @@ -484,3 +484,8 @@ int main(const int argc, const char* argv[])

>                              managedAssemblyArgv);

>      return exitCode;

>  }

> +

> +int main(const int argc, const char* argv[])

> +{

Do we want to exclude the main method when this file is included by the sos project?

—
Reply to this email directly or view it on GitHub<https://github.com/dotnet/coreclr/pull/807/files#r29096635>.